### PR TITLE
Grouping node and tx args

### PR DESCRIPTION
**Pull request by cool-ant**
Cleaning up organization args so they live where they should live and act more consistently

### CODE REVIEW

File: rust/psibase/README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+`psibase` is a library and command-line tool for interacting with psibase networks. Used by sys admins and non-Rust devs.
```
[the remark this answers is not in the document]
Comment (cool-ant) [reply rust/psibase/README.md:1]: @James-Mart That was your wording; can't say I'm clear why, other than `cargo-psibase` does the Rust side of things; I just figured `psibase` did C++ equivalents?

File: rust/psibase/src/main.rs [hunk 2]
```diff
@@ -1342,137 +1423,34 @@ pub fn parse_api_endpoint(api_str: &str) -> Result<Url, anyhow::Error> {
 #[tokio::main]
 async fn main() -> Result<(), anyhow::Error> {
     let args = parse_args()?;
-    let (client, _proxy) = build_client(&args).await?;
-    if let Some(help) = &args.help {
-        return print_help(help);
-    }
+    // if let Some(help) = &args.help {
+    //     return print_help(help);
+    // }
```
[the remark this answers is not in the document]
Comment (cool-ant) [reply rust/psibase/src/main.rs:1428]: sorry. oversight. Added back.

File: programs/psinode/tests/test_psibase.py [hunk 3]
```diff
@@ -113,12 +113,13 @@ def test_upgrade(self, cluster):
 
         with tempfile.TemporaryDirectory() as dir:
             make_package_repository(dir, [foo10])
-            a.run_psibase(['install', 'foo', '--package-source', dir])
```
[the remark this answers is not in the document]
Comment (cool-ant) [reply programs/psinode/tests/test_psibase.py:116]: a.install(...) does end up calling run_psibase(). Does that not accomplish the same thing? and if we want to run_psibase from here, where do we get the node_args from?

[the remark this answers is not in the document]
Comment (swatanabe) [reply programs/psinode/tests/test_psibase.py:116]: It does the same thing now, but I think the fact that install runs `psibase` should be an implementation detail. Doesn't `a.node_args()` work?

File: rust/psibase/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (James-Mart) [reply rust/psibase/README.md:1]: > I just figured psibase did C++ equivalents

This is the same `psibase` that you've been linking to in all of your plugins and rust services. I was just saying it's *also* used as a cli by sys admins and non-rust devs in the context of comparing it to `cargo-psibase`, which is just for rust-devs. 

I'd vote to make the description simply equal to the cli `--help` text:
```
`psibase` is a library and command-line tool for interacting with psibase networks.

See `psibase --help` for CLI usage details.
See [rust docs](docs.rs/psibase) for library usage details.
```

**Comment by James-Mart** [comment]
Is `--sign` needed for `boot`?

**Comment by cool-ant** [comment]
> Is `--sign` needed for `boot`?

@swatanabe ?

**Comment by swatanabe** [comment]
> > Is `--sign` needed for `boot`?
> 
> @swatanabe ?

No, it shouldn't be.

**Review by swatanabe — APPROVED** [review]

**Pull request merged**